WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -81,6 +82,7 @@ public static int Main(string[] args)
 				}
 				catch (OperationCanceledException ex)
 				{
+					exceptionToReport = ex;
```

Comment:
@molnard Can we clarify this? I am not aware of what should be the excepted behavior.

Since `exceptionToReport = ex` is here, it will trigger the crash reporter all the time, which is seemingly fine.
But... You mentioned before something about a feature that would bring the first instance into the front, does that work?
I tried to test it in GUI master branch, but I was able to run as many instances as I want.
I am a bit lost, to be honest...

Reply:
Yes, adding this line is wrong as well. 

Happy path:
- Starting the first instance
- Starting the second instance
- Second instance sends the "signal" to the first instance, then quits silently
- The first instance is Shown or Brought To the Front. 

In any other error situation, the crash report will show the problem.